src/treeview/TableNode.ts

```diff
@@ -2,14 +2,19 @@ import * as vscode from 'vscode';
 import { TableDetails } from '../view/app/model';
 import { INode } from './INode';
 
+export enum TableSourceEnum {
```

Comment:
Wrong name. The class name is TableNode so I suppose it's related to that. Name should be `TableNodeSourceEnum`.